packages/pytest-simcore/src/pytest_simcore/aiohttp_mockups.py

```diff
@@ -0,0 +1,56 @@
+import re
+from aioresponses.core import CallbackResult
```

Comment:
MINOR: may I suggest naming something like ``services_api_mocks_for_aiohttp_clients.py`` so I will add the equivalent with for ``httpx`` ;-)

Reply:
so long a name??? are you kidding me :rofl:

Reply:
ok, I thought you liked those :-) Then some alternative that refers to "service-API", "mocks" and "aiohttp_clients". What about creating a folder ``service_api_mocks/for_aiohttp_client.py`` and ``service_api_mocks/for_httpx_client.py,``?

Reply:
so I used the first proposal...